### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by iancmx**
Adding pull request template

### CODE REVIEW

**Review by kenzyop — APPROVED** [review]
LGTM

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation Update (if none of the other choices apply)
+
+## Checklist
+
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
+
+- [ ] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
```
Comment (AndyHsiungDev) [inline .github/pull_request_template.md:20]: Link should be updated to CONTRIBUTING.md for this repo

**Review by AndyHsiungDev — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (rocpoon) [reply .github/pull_request_template.md:20]: Is there are CONTRIBUTING.md in this repo? 😄

Comment (iancmx) [reply .github/pull_request_template.md]: Don't think there's a CONTRIBUTING.md file yet, will update the link when it exists.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+This area should provide an overview of why the work is happening and, within reason, should include relevant links to the ticket, product/design briefs, and engineering plans. We shouldn’t assume familiarity with the history of this work. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+## Types of changes
+
+What types of changes does your code ?
+_Put an `x` in the boxes that apply_
+
+- [ ] Bugfix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
```
Comment (SoJexy) [inline .github/pull_request_template.md:13]: @iancmx I think in our `Contributing` doc we wrote to email development@procurify.com if you want to introduce a breaking change. Should we change this or change our `Contributing` document?

Comment (iancmx) [reply .github/pull_request_template.md:13]: I believe that we should change this, I'll modify this checkmark to prompt the dev to email development@procurify.com if the dev wants to introduce a breaking change

**Review by SoJexy — APPROVED** [review]

**Pull request merged**